lib/views/after_auth_screens/events/event_info_page.dart

```diff
@@ -31,6 +33,16 @@ class _EventInfoPageState extends State<EventInfoPage> {
                   AppLocalizations.of(context)!
                       .strictTranslate('Event Details'),
                 ),
+                actions: [
+                  IconButton(
+                    icon: const Icon(Icons.share),
+                    onPressed: () {
+                      SocialShare.shareOptions(
+                        'https://cyberwake.github.io/applink/eventInvite?setUrl=${GraphqlConfig.orgURI}&selectOrg=${userConfig.currentOrg.id!}&eventId=${model.event.id}',
```

Comment:
What's this url?

Reply:
That's the url that is being shared which will open up the specific event.

Reply:
This is using the App link method for android and Universal Link for IOS right? because I think for us it would be better to stay away from those since we don't necessarily want to tie anything to a single domain . the deeplink/ custom link method may be better along with a generic host and scheme that doesn't link to a personal website

Reply:
This link isn't a link to a personal website it was already being used in the app for organisation invites I have just re-used that.
Here it is being used - https://github.com/PalisadoesFoundation/talawa/blob/f06a381f064818b1e8cbb72332335a9f19780287/lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart#L95

It could be easily changed to something else like `palisadoes.org` from here 
https://github.com/PalisadoesFoundation/talawa/blob/f06a381f064818b1e8cbb72332335a9f19780287/android/app/src/main/AndroidManifest.xml#L51